### ISSUE TITLE
Ver o que é essa diretiva de Upload usada anteriormente

### ISSUE THREAD

**Issue by lhas** [lhas/pep#7]

**Comment by lhas** [comment]
`
    <?= $this->Html->script('https://ajax.googleapis.com/ajax/libs/angularjs/1.4.3/angular.js') ?> 
    <?= $this->Html->script('/global/scripts/highcharts-ng.min.js'); ?>
    <?= $this->Html->script('/global/scripts/ng-file-upload-all.min.js'); ?>
    <?= $this->Html->script('/global/scripts/ng-tags-input.min.js'); ?>
    <?= $this->Html->script('/global/scripts/ui-mask/dist/mask.min.js'); ?>
    <?= $this->Html->script('/global/scripts/ui-sortable/src/sortable.js'); ?>
    <?= $this->Html->script('redepga.js') ?>
    <?= $this->Html->script('controllers.js') ?>
    <?= $this->Html->script('services.js') ?>
    <?= $this->Html->script('directives.js') ?>`

**Comment by lhas** [comment]
Problema resolvido. Era só importar o ng-file-upload e declarar no .JS principal. Tem que renomear essas paradas para PEP e ver se organizo os controllers, factories, etc, por arquivo.
